src/fw/drivers/display/display.h

```diff
@@ -17,6 +17,7 @@
 #pragma once
 
 #include "board/display.h"
+#include "applib/graphics/gtypes.h"
```

Comment:
Why is applib being referenced from the display driver? That doesn't seem right.

Reply:
so it looks like display.h does `typedef struct GPoint GPoint;`, but `struct GPoint` is not defined in that header. other code likely works because whoever is including this header includes the other first somehow?

Reply:
Ah, that sounds like late hacks for PT2 that never got cleaned up...